src/scripts/main.js

```diff
@@ -1,3 +1,30 @@
 'use strict';
 
-// write your code here
+const numbers = document.querySelectorAll('.population');
+
+const populations = [...numbers].map((el) => el.innerText);
+
+const separatedNumbers = populations.join('/');
```

Comment:
```suggestion
const numbers = document.querySelectorAll('.population');
const populations = [...numbers].map((el) => el.innerText);
const separatedNumbers = populations.join('/');
```

Reply:
no need to make so many empty lines. but it isn't a mistake, just a suggestion

Reply:
Thanks) More elegant and twice shorter) The logic is the same though.